tests/common/devices.py

```diff
@@ -312,6 +312,28 @@ def get_networking_uptime(self):
             self.logger.error("Exception raised while getting networking restart time: %s" % repr(e))
             return None
 
+    def get_image_info(self):
+        """
+        @summary: get list of images installed on the dut.
+                  return a dictionary of "current, next, installed_list"
+        """
+        output = self.command("sonic_installer list")["stdout"]
+        lines  = output.split('\n')
+        ret    = {}
+        list   = []
+        for line in lines:
```

Comment:
Python has a `splitlines()` method for this, so you can do:
```suggestion
        for line in output.splitlines():
```

Reply:
thanks. done!